package.json

```diff
@@ -8,18 +8,21 @@
 	"main": "dist/pacman-contribution-graph.min.js",
 	"scripts": {
 		"build": "webpack --config webpack.prod.js",
-		"dev": "webpack --progress --config webpack.dev.js",
+		"start": "node dist/index.js --username=diogocarrola --output=svg > assets/pacman-contributions.svg",
+		"dev": "webpack serve --config webpack.dev.js",
 		"prepublishOnly": "npm run build",
 		"build-action": "pnpm run --prefix ./github-action build"
 	},
 	"devDependencies": {
+		"html-webpack-plugin": "^5.6.3",
```

Comment:
Why do we need `html-webpack-plugin` & `webpack-dev-server` for? We can export svg directly I guess.